src/lib/zkapp.ts

```diff
@@ -611,6 +611,50 @@ class SmartContract {
     party.body.events = Events.pushEvent(party.body.events, eventFields);
   }
 
+  fetchEvents(
+    start: UInt32,
```

Comment:
could we also make `start` optional, and default to `0`? David suggested that on the events RFC as a common use case